contrib/hadoop-store-builder/src/java/voldemort/store/readonly/fetcher/HdfsFetcher.java

```diff
@@ -76,16 +78,11 @@
      * {@link AdminServiceRequestHandler#setFetcherClass(voldemort.server.VoldemortConfig)}
      */
     public HdfsFetcher(VoldemortConfig config) {
-        this(config,
-             config.getReadOnlyFetcherMaxBytesPerSecond(),
-             config.getReadOnlyFetcherReportingIntervalBytes(),
-             config.getReadOnlyFetcherThrottlerInterval(),
-             config.getFetcherBufferSize(),
-             config.getReadOnlyFetchRetryCount(),
-             config.getReadOnlyFetchRetryDelayMs(),
-             config.isReadOnlyStatsFileEnabled(),
-             config.getReadOnlyMaxVersionsStatsFile(),
-             config.getFetcherSocketTimeout());
+        this(config, config.getReadOnlyFetcherMaxBytesPerSecond(), config.getReadOnlyFetcherReportingIntervalBytes(),
```

Comment:
Can we try to avoid unnecessary formatting changes please? I can export my IntelliJ editor settings if you'd like (: